sdk/storage/README.md

```diff
@@ -1,16 +1,21 @@
-# Azure Storage SDK V10 for JavaScript
+# Azure Storage client library for JavaScript
```

Comment:
Seems upper case for 1st char is more sutiable for title, like "Azure Storage Client Library for JavaScript"

Reply:
The template calls out that headers should use **sentence-style capitalization**
https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md

Reply:
Ok to keep align cross all Azure SDKs.